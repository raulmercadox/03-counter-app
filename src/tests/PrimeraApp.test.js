import React from "react";
import { shallow } from "enzyme";
import PrimeraApp from "../PrimeraApp";

describe("Pruebas en <PrimeraApp />", () => {
  test("debe de mostrar </PrimeraApp /> correctamente", () => {
    const saludo = "Hola, mi nombre es Raul Mercado";
    const wrapper = shallow(<PrimeraApp saludo={saludo} />);

    expect(wrapper).toMatchSnapshot();
  });

  test("debe de enviar el subtitulo enviado por props", () => {
    const saludo = "Hola, mi nombre es Raul Mercado";
    const subTitulo = "soy un subtitulo";
    const wrapper = shallow(
      <PrimeraApp saludo={saludo} subtitulo={subTitulo} />
    );

    const textoParrafo = wrapper.find("p").text();
    // console.log(textoParrafo);
    expect(textoParrafo).toBe(subTitulo);
  });
});
