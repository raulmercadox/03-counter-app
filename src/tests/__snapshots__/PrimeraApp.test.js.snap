// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en <PrimeraApp /> debe de mostrar </PrimeraApp /> correctamente 1`] = `
<Fragment>
  <h1>
    Hola, mi nombre es Raul Mercado
    !!!
  </h1>
  <p>
    soy un subtitulo
  </p>
</Fragment>
`;
